aggregator/src/bin/aggregator.rs

```diff
@@ -145,6 +143,39 @@ async fn main() -> Result<()> {
     .await
 }
 
+fn build_aggregator_api_handler<'a>(
+    options: &Options,
+    config: &'a Config,
+    datastore: &Arc<Datastore<RealClock>>,
+) -> anyhow::Result<Option<(impl Handler, &'a AggregatorApi)>> {
+    let Some(aggregator_api) = &config.aggregator_api else {
+        return Ok(None);
+    };
```

Comment:
Does this work on Rust 1.65, currently Janus' MSRV? I don't think bumping MSRV is a dealbreaker, FWIW.

Reply:
https://doc.rust-lang.org/rust-by-example/flow_control/let_else.html stable since 1.65